src/context.rs

```diff
@@ -604,6 +614,9 @@ impl<'a> EventCx<'a> {
         }
     }
 
+    /// Used to determine if you should send an event to another view. This is basically a check for pointer events to see if the pointer is inside a child view and to make sure the current view isn't hidden or disabled.
+    /// Usually this is used if you want to propagate an event to a child view
+    // TODO: should we make this public so that it can be used in custom components? It's not a public function atm but I see it used in all of the container-like `View`s.
     pub(crate) fn should_send(&mut self, id: Id, event: &Event) -> bool {
```

Comment:
should we make this public so that it can be used in custom components? It's not a public function atm but I see it used in all of the container-like `View`s.

Reply:
+1 for this. I've resorted to adding event listeners for all of the pointer events that just return true to stop the propagation of events

Reply:
I made it public. @dzhou121 to confirm